core/src/main/java/io/opentelemetry/android/features/diskbuffering/DiskBufferingConfiguration.java

```diff
@@ -70,12 +72,30 @@ public static final class Builder {
         private long maxFileAgeForWriteMillis = TimeUnit.SECONDS.toMillis(30);
         private long minFileAgeForReadMillis = TimeUnit.SECONDS.toMillis(33);
         private long maxFileAgeForReadMillis = TimeUnit.HOURS.toMillis(18);
+        private boolean enableDebugMode = false;
 
         private ExportScheduleHandler exportScheduleHandler = DefaultExportScheduleHandler.create();
 
+        /** Sets the debug mode for disk buffering, enabling additional logging. */
+        public Builder setDebugMode(boolean enableDebugMode) {
+            this.enableDebugMode = enableDebugMode;
+            Logger.getLogger(DiskBufferingConfiguration.class.getName())
+                    .log(
+                            Level.INFO,
+                            "Disk buffering has been "
+                                    + (enabled ? "enabled." : "disabled.")
+                                    + (enableDebugMode ? " Debug mode is active." : ""));
```

Comment:
I don't think we need to log the `enabled` setting here -- that's a separate concern below. I recommend changing the log message to be simpler and only reflect the new value of `enableDebugMode`.